lib/matplotlib/tests/test_axes.py

```diff
@@ -4335,6 +4335,17 @@ def test_pandas_indexing_hist():
     fig, axes = plt.subplots()
     axes.hist(ser_2)
 
+@cleanup
+def test_axis_set_tick_params_labelsize_labelcolor():
+    # Tests fix for issue 4346
+    axis_1 = plt.subplot()
+    axis_1.yaxis.set_tick_params(labelsize=30, labelcolor='red', direction='out')
+
+    # Expected values after setting the ticks
+    assert axis_1.yaxis.majorTicks[0]._size == 4.0
```

Comment:
I can't seem to find a way to get either of labelsize, labelcolor, size, or color out of the public API.  There is a [get_markersize](http://matplotlib.org/api/lines_api.html#matplotlib.lines.Line2D.get_markersize) in the docs that would return _size,  however I don't see it in the code. Should I implement this?